From 10c7f13a3d868d489054d8af4463bb68036055c6 Mon Sep 17 00:00:00 2001
From: Damian Hobson-Garcia <dhobsong@igel.co.jp>
Date: Tue, 26 Oct 2021 15:05:38 +0900
Subject: [PATCH 2/3] waylandws_client: Rename DRM device setup function

Make the function name better reflect its purpose. Namely,
getting the drm device and setting up authentication if
necessary.
---
 src/waylandws_client.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/waylandws_client.c b/src/waylandws_client.c
index 20bfac9..0638862 100644
--- a/src/waylandws_client.c
+++ b/src/waylandws_client.c
@@ -681,7 +681,7 @@ static bool authenticate_kms_device(WLWSClientDisplay *display)
 	return true;
 }
 
-static bool setup_buffer_sharing(WLWSClientDisplay *display)
+static bool setup_drm_device(WLWSClientDisplay *display)
 {
 	display->fd = -1;
 
@@ -764,7 +764,7 @@ static WSEGLError WSEGLc_InitialiseDisplay(NativeDisplayType hNativeDisplay,
 	wl_registry_add_listener(display->wl_registry, &wayland_registry_listener, display);
 
 	/* Now setup the DRM device */
-	if (!setup_buffer_sharing(display)) {
+	if (!setup_drm_device(display)) {
 		err = WSEGL_BAD_NATIVE_DISPLAY;
 		goto fail;
 	}
-- 
2.25.1

